source/03-components/modal/modal.es6.js

```diff
@@ -0,0 +1,112 @@
+import Drupal from 'drupal';
+import once from 'once';
+import KEYCODE from '../../00-config/_KEYCODE.es6';
+
+Drupal.behaviors.accordion = {
```

Comment:
This should be renamed to `modal`.

Reply:
New convention proposal: we prefix our behavior names with gesso, e.g. `Drupal.behaviors.gessoModal`. It occurs to me that `accordion` and `modal` are common enough terms that it's possible some other module could also define `Drupal.behaviors.modal`.

Reply:
Yeah, I like that idea @kmonahan.
Also, nice catch @dcmouyard , I have a bad habit of copy/pasting the first few lines of another JS and forgetting to rename them 🤦

Reply:
Me too! 😄